### ISSUE TITLE
Missing whitespace surrounding operators in CSS calc() causing incorrect results of calculations

### ISSUE THREAD

**Issue by JamesDPC** [digitalnsw/nsw-design-system#229]
Hi all

In the hero banner component there is no whitespace surrounding the "+" operator for calc(), resulting in unexpected results.
This possibly exists in other components as well ? When I add whitespace around the operators here, the results match the docs at https://digitalnsw.github.io/nsw-design-system/components/hero-banner/blank.html

https://github.com/digitalnsw/nsw-design-system/blob/master/src/components/hero-banner/_hero-banner.scss#L229
https://github.com/digitalnsw/nsw-design-system/blob/master/src/components/hero-banner/_hero-banner.scss#L137

Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/calc#notes

>The + and - operators must be surrounded by whitespace

Example rule:

```
padding-left: calc(var(--nsw-container-width)/4+#{rem($grid-gutters)}*2);
```

In the docs example using scss, this results in:

```css
padding-left: calc(var(--nsw-container-width)/4 + 1rem);
```

When built, though, the + is treated as a concatenator and the result is being interpreted as:

```css
padding-left: calc(var(--nsw-container-width)/40.5rem*2);
```
The 4 and the 0.5rem are being concatenated together, the 0.5 comes from rem(8px)

Cheers
James

**Comment by anna-yeshtukova** [comment]
Hi James. Thank you for bringing it to our attention. We will try to work on it shortly and will keep you posted.

**Comment by anna-yeshtukova** [comment]
Hi @JamesDPC ,

We had looked at this issue and I can confirm that our SCSS compiler was successfully fixing it so there was never interpretation issue for end user. However we fixed the source code in our last 3.2.2 release and confirmed that it is correct across all other components.
Once again thank you for bringing it to our attention!

**Comment by JamesDPC** [comment]
Thanks for updating on this, the change in 3.2.2 looks correct.

I took a further look in < 3.2.2 and yeah it looks like cssnano is cleaning up the scss with the missing whitespace. I was looking at an unminified build I'd created for development which wasn't using cssnano, so it just used the spacing from the source scss :D
